extensions/vault/deployment/pom.xml

```diff
@@ -0,0 +1,99 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>quarkus-vault-parent</artifactId>
+        <groupId>io.quarkus</groupId>
+        <version>999-SNAPSHOT</version>
+        <relativePath>../</relativePath>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>quarkus-vault-deployment</artifactId>
+    <name>Quarkus - Vault - Deployment</name>
+
+    <dependencies>
+
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-vault</artifactId>
+        </dependency>
+
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-core-deployment</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-arc-deployment</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-jackson-deployment</artifactId>
```

Comment:
We have a strong dependency to Jackson?

Reply:
not anymore. at some point I was struggling to get the native build to pass. not getting into the specifics discussed in https://github.com/quarkusio/quarkus/pull/2897#issuecomment-523775095. but later on we changed the design for a spi approach https://github.com/quarkusio/quarkus/pull/2897#issuecomment-527574691.
and I think that was my issue, not jsonb.
anyway I switched back to jsonb, and everything seems to work fine (waiting for the CI to confirm).